packages/yew-bootstrap/src/component/pagination.rs

```diff
@@ -0,0 +1,95 @@
+use yew::prelude::*;
+
+
+#[derive(Clone, PartialEq, Eq)]
+pub enum PaginationSize {
+    Large,
+    Normal,
+    Small,
+}
+
+impl Default for PaginationSize {
+    fn default() -> Self {
+        PaginationSize::Normal
+    }
+}
+
+pub struct Pagination {}
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Props {
+    #[prop_or_default]
+    pub callback: Callback<u32>,
+
+    #[prop_or_default]
+    pub current_page: u32,
+
+    #[prop_or_default]
+    pub max_items_per_page: u32,
+
+    #[prop_or_default]
+    pub size: PaginationSize,
+
+    #[prop_or_default]
+    pub total_page_count: u32,
+}
+
+#[function_component(ListPagination)]
+pub fn list_pagination(props: &Props) -> Html {
+    if props.total_page_count < props.max_items_per_page {
+        return html! {};
+    }
+
+    let max_page = (props.total_page_count as f32 / props.max_items_per_page as f32).ceil() as u32;
+    let mut pages: Vec<u32> = vec![];
+    for page in 0..max_page {
+        pages.push(page);
+    }
```

Comment:
You should be able to do this iteration in the html over the range, rather than pushing numbers into a vec.